core/com.b2international.snowowl.core.rest.tests/src/com/b2international/snowowl/core/rest/resource/ResourceApiTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2011-2021 B2i Healthcare Pte Ltd, http://b2i.sg
```

Comment:
For new files use only the current start year.

Reply:
Done